packages/analytics-js-plugins/src/consentManager/index.ts

```diff
@@ -9,6 +15,30 @@ const ConsentManager = (): ExtensionPlugin => ({
   initialize: (state: ApplicationState) => {
     state.plugins.loadedPlugins.value = [...state.plugins.loadedPlugins.value, pluginName];
   },
+  consentManager: {
+    init(
+      state: ApplicationState,
+      pluginsManager: IPluginsManager,
+      selectedConsentManager: string,
+      logger?: ILogger,
+    ): void {
+      logger?.debug('ConsentManager initialization');
+
+      if (selectedConsentManager) {
+        // Initialize selected consent manager and get the consent info
+        const { consentManagerInitialized, allowedConsentIds, deniedConsentIds } =
+          pluginsManager.invokeSingle(`${selectedConsentManager}.init`, logger);
```

Comment:
The extension point should be named generic. As the necessary plugin (as selected by the user) is already initialized, invoking the plugin generically would be sufficient.

Reply:
Do you mean `selectedConsentManager ` is enough, no need for the `.init` extension point?

Reply:
No need for `selectConsentManger` input here.
The extension point prefix can be named like `customConsentManager`. So, you'll invoke it as `customConsentManager.init`.

Reply:
Addressed